packages/manager/apps/web/client/app/domain/domains.routing.js

```diff
@@ -75,9 +76,17 @@ export default /* @ngInject */ ($stateProvider) => {
           },
         },
       }),
-      goToRestoreRenew: /* @ngInject */ ($state) => (domains) =>
-        $state.go('app.domain.index.restore-renew', { domains }),
+      goToRestoreRenew: /* @ngInject */ ($state, atInternet) => (domains) => {
+        atInternet.trackClick({
+          name: 'web::domain::::page::button::renew-restore::domain',
+          type: 'action',
+        });
```

Comment:
```suggestion
```
same https://github.com/ovh/manager/pull/13862#discussion_r1824031932

Reply:
This one has been removed with requester